doc/source/admin/scaling.md

```diff
@@ -566,221 +394,102 @@ $ ./scripts/galaxy-main -c config/galaxy.yml --server-name handler2 --attach-to-
+<ScheduleEntry: celery.backend_cleanup celery.backend_cleanup() <crontab: 0 4 * * * (m/h/d/dM/MY)>
+[2022-02-17 16:31:26,824: DEBUG/MainProcess] beat: Ticking with max interval->5.00 minutes
+[2022-02-17 16:31:26,825: DEBUG/MainProcess] beat: Waking up in 5.00 minutes.
+```
+
+More advanced logging options are described in the Galaxy [Logging Configuration documentation](config_logging)
 
 ## Starting and Stopping
 
-If you are using the **uWSGI + Webless job handlers** deployment strategy or want to run your Galaxy server as a
-persistent service, you can control it through a process manager. The current recommended process manager is
-[Supervisord](http://supervisord.org/). If you are comfortable with systemd and are running a relatively modern Linux
-distribution, you can also configure Galaxy as a service directly in systemd.
-
-### Supervisord
-
-You can use a supervisord config file like the following or be inspired by [this
-example](https://github.com/galaxyproject/galaxy/blob/dev/contrib/galaxy_supervisor.conf). Be sure to `supervisord
-update` or `supervisord reread && supervisord restart` whenever you make configuration changes.
-
-```ini
-[program:web]
-command         = /srv/galaxy/venv/bin/uwsgi --yaml /srv/galaxy/config/galaxy.yml
-directory       = /srv/galaxy/server
-umask           = 022
-autostart       = true
-autorestart     = true
-startsecs       = 10
-stopwaitsecs    = 65
-user            = galaxy
-numprocs        = 1
-stopsignal      = INT
-```
-
-This configuration defines a "program" named `web` which represents our Galaxy uWSGI frontend. You'll notice that we've
-set a command, a directory, a umask, all of which you should be familiar with. Additionally we've specified that the
-process should `autostart` on boot, and `autorestart` if it ever crashes. We specify `startsecs` to say "the process
-must stay up for this long before we consider it OK. If the process crashes sooner than that (e.g. bad changes you've
-made to your local installation) supervisord will try again a couple of times to restart the process before giving up
-and marking it as failed. This is one of the many ways supervisord is much friendly for managing these sorts of tasks. 
-
-The value of `stopwaitsecs` should be at least as large as the smallest value of uWSGI's `reload-mercy`,
-`worker-reload-mercy`, and `mule-reload-mercy` options, all of which default to `60`.
+If you want to run your Galaxy server as a persistent service, you can include the `galaxy` script from Galaxy's
+virtualenv in your process manager configuration. You can then continue using the `galaxyctl` command as usual
+to start/stop/restart Galaxy or follow the logs.
 
-If using the **uWSGI + Webless** strategy, you'll need to addtionally define job handlers to start. There's no simple
-way to activate a virtualenv when using supervisor, but you can simulate the effects by setting `$PATH` and
-`$VIRTUAL_ENV`:
+## Transparent restarts
```

Comment:
Maybe move this section after `### Systemd` ? Also, do you want it to be under `## Starting and Stopping`? If yes, you need an additional `#`.

Reply:
```suggestion
### Transparent restarts
```